### ISSUE TITLE
documentation url

### ISSUE THREAD

**Issue by JonathanRoth13** [smarnach/pyexiftool#36]
the url for the documentation should read https://smarnach.github.io/pyexiftool/ as opposed to https://smarnach.github.com/pyexiftool/

thank you for creating such a good tool

**Comment by sylikc** [comment]
Yeah, this project hasn't been updated in awhile.  My fork has some bug fixes and the right documentation link, I have some issues out that need documentation update... would you like to help? 😊 https://github.com/sylikc/pyexiftool/issues/28

**Comment by JonathanRoth13** [comment]
Kevin,

I would like to help with this project, however I am unavailable for the moment. I should be less busy in roughly two months. I'll get back to you then.

Regards,
Jonathan Roth

------- Original Message -------

**Comment by sylikc** [comment]
Sure thing.  To follow the latest development, it's currently on this branch https://github.com/sylikc/pyexiftool/tree/v0.5.x-py3-refactor ... "maybe" just maybe by the time you're available, I'll have merged that into the master branch.

Anyhow, I've been trying to figure out Sphinx and autogenerating docs from the docstrings.  Feel free to comment on the docs issue https://github.com/sylikc/pyexiftool/issues/28 or create a new one to discuss there.

**Comment by sylikc** [comment]
@JonathanRoth13 I've updated my latest version to the refactored Python3 version v0.5.0.  Should be easier to use, while some features have been jumbled around a bit.  Can you review the newly updated documentation and see if it is coherent? :D 

https://sylikc.github.io/pyexiftool/

(Please review and discuss on that issue thread if you can https://github.com/sylikc/pyexiftool/issues/28)
